.ci/close-create-milestone.sh

```diff
@@ -7,12 +7,12 @@ source ./.ci/util.sh
 checkForVariable "GITHUB_TOKEN"
 
 echo "Close previous milestone at github"
-MILESTONE_ID=$(curl -s \
+MILESTONE_NUMBER=$(curl -s \
```

Comment:
`MILESTONE_NUMBER` is more-accurate since `id` is also present in the response from GitHub and it represents something else.